.github/workflows/decline-template.yml

```diff
@@ -0,0 +1,124 @@
+    if: github.event_name == 'issue_comment' && github.event.comment.body == '/decline' && (contains(github.event.issue.labels.*.name, 'add-template') || contains(github.event.issue.labels.*.name, 'update-template'))
+    outputs:
+      github-link: ${{ steps.parse-issue-form-body.outputs.github-link }}
+      npm-package: ${{ steps.parse-issue-form-body.outputs.npm-package }}
+      error: ${{ steps.parse-issue-form-body.outputs.error }}
+      request-type: ${{ steps.get-type.outputs.REQUEST_TYPE }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: npm install
+      - name: Get type of request
+        id: get-type
+        run: |
+          if ${{ contains(github.event.issue.labels.*.name, 'add-template') }}
+          then
+            echo '::set-output name=REQUEST_TYPE::review-request'
+          fi
+          if ${{ contains(github.event.issue.labels.*.name, 'update-template') }}
+          then
+            echo '::set-output name=REQUEST_TYPE::update-request'
+          fi
+      - name: Parse issue form body
+        id: parse-issue-form-body
+        run: |
+          node src/parse-issue-form-body.js ${{ steps.get-type.outputs.REQUEST_TYPE }}
+        env:
+          GITHUB_ISSUE_PAYLOAD: ${{ github.event.issue.body }}
+  set-owner-repo:
+    name: Set the owner and repo name.
+    runs-on: ubuntu-latest
+    needs: [parse-issue-form-body]
+    outputs:
+      github-repo-owner: ${{ steps.set-values.outputs.github-repo-owner }}
+      github-repo: ${{ steps.set-values.outputs.github-repo }}
+    steps:
+      - name: Set the owner and repo name.
+        id: set-values
+        run: |
+          ownerRepo='${{ github.repository }}'
+          owner='${{ github.repository_owner }}'
+          echo "::set-output name=github-repo-owner::$owner"
+          github_repo=${ownerRepo#"$owner/"}
+          echo "::set-output name=github-repo::$github_repo"
+  check-permissions:
+    needs: [parse-issue-form-body, set-owner-repo]
+    uses: ./.github/workflows/check-permissions-workflow.yml
+    secrets:
+      ALLOWLIST_ADMINS: ${{ secrets.ALLOWLIST_ADMINS }}
+    with:
+      user-login: ${{ github.event.issue.user.login }}
+      npm-package: ${{ needs.parse-issue-form-body.outputs.npm-package }}
+      github-repo-owner: ${{ needs.set-owner-repo.outputs.github-repo-owner }}
+      github-repo: ${{ needs.set-owner-repo.outputs.github-repo }}
+  update-request-close-issue:
+    name: Close issue if approval declined on update request
+    runs-on: ubuntu-latest
+    needs: [parse-issue-form-body, check-permissions]
+    if: ${{ always() && needs.parse-issue-form-body.result == 'success' && needs.parse-issue-form-body.outputs.request-type == 'update-request' && needs.check-permissions.outputs.is-admin == 'true' }}
+    steps:
+      - uses: peter-evans/close-issue@v1
+        with:
+          comment: "Full approval not granted. Closing issue..."
```

Comment:
@AmyJZhao , so the status will remain "InVerification"? https://github.com/adobe/aio-template-submission/pull/273/files#diff-d7a677c9ed6b55c54b4c54d6de22d4c5832d355a6c9a22ffd98de08932aa8074R88
it yes, then this is not probably that we want, at least we shouldn't close the issue and let a developer note about rejection

Reply:
Updated job so that status will be set to Rejected.